src/adagrams.js

```diff
@@ -1,15 +1,168 @@
+const letterPool = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const scoreChart = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10,
+};
+
+//creates string of all letters in letter pool by distribution
+const getLetters = () => {
+  let multiStr = "";
+  for (const letter in letterPool) {
+    multiStr += letter.repeat(letterPool[letter]);
+  }
+  return multiStr;
+};
+//const letterBank = getLetters();
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const randomHand = [];
+  let letterBank = getLetters();
+  for (let i = 0; i < 10; i++) {
+    let randomLetter =
+      letterBank[Math.floor(Math.random() * letterBank.length)];
+    randomHand.push(randomLetter);
+    letterBank = letterBank.replace(randomLetter, "");
+  }
+  return randomHand;
 };
 
 export const usesAvailableLetters = (input, lettersInHand) => {
-  // Implement this method for wave 2
+  let filteredHand = lettersInHand;
+
+  for (let i = 0; i < input.length; i++) {
+    console.log(input[i]);
+
+    if (filteredHand.includes(input[i]) === false) {
+      return false;
+    } else if (filteredHand.includes(input[i]) === true) {
```

Comment:
Same idea as before, but also, since there are only two possibilities here (The hand includes the input or it doesn't) we don't need to make it an else if statement. Instead, we can just make this an else.